### ISSUE TITLE
 "displayMode: taglist" problem with multisite

### ISSUE THREAD

**Issue by get-web** [octobercms/october#5826]
Current Build 3.7.4
I created tags for blog posts through tailor
### blog/post.yaml:
```
    tags:
          tab: Manage
          label: Tags
          type: entries
          displayMode: taglist
          source: Blog\Tag
```
I save one or more tags (it doesn't matter). Then I switch to another language and just click "Save". I come back and the tags disappear. 

If I use any other type of displayMode. For example, relation, then the tags are saved.

It seems to have started when I upgraded from version 3.6 to 3.7

**Comment by daftspunk** [comment]
Hey @get-web 

Thanks for the report. I saw this before but couldn't reproduce it. I also can't reproduce it today (https://github.com/octobercms/test-plugin/commit/3602c3f89121edac8182b028b2e4491bf8eef4da).

Can you give me some more detail, with step by step instructions, perhaps record a video?

Also, what is `multisite` property set to for the Tag and the Blog Post blueprints?

**Comment by get-web** [comment]
https://www.youtube.com/watch?v=gfgCTnO3Uz4

**blog/post.yaml:**

```
handle: Blog\Post
type: stream
name: Post
drafts: true
multisite: true

customMessages:
    buttonCreate: New Post

primaryNavigation:
    label: Blog
    icon: octo-icon-file
    iconSvg: modules/tailor/assets/images/blog-icon.svg
    order: 95

navigation:
    label: Posts
    icon: octo-icon-pencil
    order: 100

fields:
    banner:
        tab: Manage
        label: Banner
        type: fileupload
        mode: image
        maxFiles: 1
        translatable: false

    tags:
          tab: Manage
          label: Tags
          type: entries
          displayMode: taglist
          source: Blog\Tag
          
    categories:
        tab: Manage
        label: Categories
        commentAbove: 'Select categories the blog post belongs to'
        type: entries
        source: Blog\Category

    summary:
        tab: Summary
        label: Summary Text
        type: textarea
        size: small

    views:
        tab: System
        label: Views
        type: number
        default: 0
        span: auto
        hidden: true
        
    block_builder:
        type: mixin
        source: Fields\Blocks
        
    seo:
        type: mixin
        source: Seo\Fields
```


**blog/tag.yaml**
```
handle: Blog\Tag
type: structure
name: Tag
drafts: false
multisite: true

customMessages:
    buttonCreate: New Tag

structure:
    maxDepth: 1

navigation:
    label: Tags
    icon: icon-tree
    parent: Blog\Post
    order: 200

fields:
    title: 
        placeholder: Tag name
    
    seo:
        type: mixin
        source: Seo\Fields
```

**Comment by get-web** [comment]
By the way. I forgot to clarify, if I add `translatable: false` to the tags, then the problem still remains
```
    tags:
          tab: Manage
          label: Tags
          type: entries
          displayMode: taglist
          source: Blog\Tag
          translatable: false
```

**Comment by daftspunk** [comment]
This should be fixed in v3.7.5

Please test it again, hopefully it works.
